packages/common/driver-definitions/src/storage.ts

```diff
@@ -385,6 +385,32 @@ export interface IDocumentServiceFactory {
 	): Promise<IDocumentService>;
 }
 
+export interface ISocketStorageDiscoveryParts {
+	/**
+	 * Niche interface to allow odsp-driver to expose specific pieces necessary for /broadcast-signal
+	 * endpoint. Please do not use this for anything else. We will phase this out once we know how
+	 * the interface for a more uniform interface across multiple drivers looks.
+	 */
```

Comment:
This seems like it should be the doc string for the first hole class, maybe with a remarks/privateRemarks section.

Reply:
I don't understand this bit: ` for the first hole class, ` <-- did you mean something different?

Reply:
Yeah, @alexvy86 is correct; not sure how I produced "first hole class" from "whole class".